config/locales/en.yml

```diff
@@ -62,4 +62,9 @@ en:
           past_date: You specified a "Not needed after" date that is in the past. "Not needed after" date must be in the future.
           pickup_choose_text: --Please choose a pickup location--
           success_html: <span class="font-weight-bold">Success!</span> "%{bib_summary}" pickup location was updated.<br>
-          error_html: <span class="font-weight-bold">Sorry!</span> Something went wrong and "%{bib_summary}" pickup location was not updated. Contact your campus library if you need assistance.<br>
\ No newline at end of file
+          error_html: <span class="font-weight-bold">Sorry!</span> Something went wrong and "%{bib_summary}" pickup location was not updated. Contact your campus library if you need assistance.<br>
+        place_hold:
+          missing_params: To place a hold "Pickup At library and "Not Needed After" date must be filled in.
+          select_volumes: Please select a volume to place a hold.
```

Comment:
We'll want to have this and other new stuff to be a part of the label review (i.e. #2)

Reply:
Might also be a good idea to review this in real time with Ruth to go through all of the scenarios that generate these messages.